packages/@okta/vuepress-site/docs/guides/request-user-consent/require-consent/index.md

```diff
@@ -35,11 +35,15 @@ Use the following steps to display the user consent dialog box as part of an Ope
 
 The following section provides example requests for enabling the consent dialog box using the APIs. You must first set the `consent_method` property and then enable consent for the scope.
 
-> **Note:** See the [Settings table in the Apps API doc](/docs/reference/api/apps/#settings-8) for more information on this parameter.
+A consent dialog appears depending on the values of three elements:
+
+* `prompt`: a query parameter used in requests to `/oauth2/${authServerId}/v1/authorize`(custom authorization server) or `/oauth2/v1/authorize` (Org authorization server)
```

Comment:
should we link to /docs/reference/api/oidc/#parameter-details for the param details?